docs/app/Examples/views/Card/Content/ExtraContent.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Card, Icon } from 'stardust'
+
+const description = ['Amy',
+  'is',
+  'a',
+  'violinist',
+  'with',
+  '2',
+  'years',
+  'experience',
+  'in',
+  'the',
+  'wedding',
+  'industry.',
+  'She',
+  'enjoys',
+  'the',
+  'outdoors',
+  'and',
+  'currently',
+  'resides',
+  'in',
+  'upstate',
+  'New',
+  'York.'].join(' ')
```

Comment:
This strikes me as strange, can we use a regular string here?  If we're fighting max column issues, we can put more content on a single line:

```jsx
const description = [
  'Amy is a violinist with 2 years experience in the wedding industry.',
  'She enjoys the outdoors and currently resides in upstate New York.',
].join(' ')
```